mantidproject/mantid
# Add case for sqw workspaces in show data

Currently if a workspace had a numeric axis (e.g. sqw workspaces) when showing data workbench would list the lower bin edge on the vertical header and whould not show the unit.
This PR add a case for numeric axies when generating the virtical header so that is both uses the bin center and unit.
 
**Report to:** SpencerHowells

**To test:**
1. open workbench
1. load iris26176_graphite002_sqw.nxs (download from the comment by SpencerHowells on the origional issue)
1. show data from workspace
1. the virtical header should have units and show the bin center (0.55, 0.65, 0.75....)
1. load any other non numeric axis workspace.
1. show data
1. it should appear as it would normaly.

Fixes #27013 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
